UPGRADE-3.x.md

```diff
@@ -4,6 +4,17 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
```

Comment:
Doing a release on 3.x first could avoid to have to remerge again manually after the release

Reply:
Yes, I know, but my plan is:
1. Merge this one that is the only one required on 4.x to continue working
2. Approve @phansys PRs
3. @phansys will merge them, release it, and merge to 4.x

Otherwise we will end up with 2 releases in linke 1,2 days, and I prefer to not delay this further.